### ISSUE TITLE
Disabled service still active on poller reload, only poller restart really disables it

### ISSUE THREAD

**Issue by btassite** [centreon/centreon-engine#67]
Centreon 2.7.7

```
$ mysql -t centreon_storage -e "select enabled from services where services.service_id=3346;"
+---------+
| enabled |
+---------+
|       1 |
+---------+
```

In the web interface under Configuration/Services: disabled service by clicking on the red crossed out circle on the right of the service, service line becomes grey (and circle becomes green checkmark), applied configuration for poller with Restart Monitoring Engine "Reload" option (Move Export Files also checked). Service is grey in configuration but is still listed in monitoring view and the above query returns the same result. Under Configuration/Services/Services by host, Service Extended Info, Status shows as Disabled. Under Administratios/Logs, service shows with the proper datestamp as Disabled (in purple).

The above query returns a 0 and the service check disappears from monitoring view only after using Restart Monitoring Engine with "**Restart**" option instead of "**Reload**".

Can't see anything relating to this in any of the logs (/var/log/centreon-engine/centengine.debug, centengine.log, init.debug). One possibly pertinent piece of information: the service was originally duplicated from a different host and renamed about 2 months ago (and has been working fine sine). I have seen some strange behaviour when using duplication (see escalation/dependencies bug, possibly due to initially having the same host IP), but this is just a thought to be entertained.

**Comment by ganoze** [comment]
Hi and thank you for your feedback.

Are you using the latest versions of Centreon Engine (1.5.3) and Centreon Broker (2.11.7) for Centreon 2.7 ?

**Comment by btassite** [comment]
Centreon Engine 1.5.1
Centreon Broker 2.11.5

**Comment by btassite** [comment]
Do you need more information or has this been reproduced and/or fixed it in a new release?

**Comment by ganoze** [comment]
Yes, could you check that this behavior is still existing with latest bugfixes of Centreon Engine (1.5.3) and Centreon Broker (2.11.8) for Centreon 2.7 ?

**Comment by btassite** [comment]
Testing a new version is planned but the schedule is a moving target. I'm not sure I will be able to reproduce it though as I still think it has to do with db/cleanup inconsistency following duplicated hosts.
